README.md

```diff
@@ -0,0 +1,21 @@
+# For common users:
+Set the needed variables in .env (take librephotos.env as model)
+Just pull the repo and run
```

Comment:
```suggestion
Clone the repo:

```shell
git clone git@github.com:LibrePhotos/librephotos-docker.git
```

Reply:
Right, added it